wrappers/javascript/anoncreds-nodejs/test/bindings.test.ts

```diff
@@ -8,7 +8,7 @@ describe('bindings', () => {
   test('version', () => {
     const version = anoncreds.version()
 
-    expect(version).toEqual('0.1.0-dev.4')
+    expect(version).toEqual('0.1.0-dev.5')
```

Comment:
This should probably be removed later on :).

Reply:
Or changed to match a regex